subsys/zigbee/Kconfig

```diff
@@ -200,6 +206,33 @@ config ZIGBEE_UART_TX_BUF_LEN
 
 endif #ZIGBEE_HAVE_SERIAL
 
+menuconfig ZIGBEE_CLI_SHELL
+	bool "Enable Zigbee CLI Shell"
+	select SHELL
+
+if ZIGBEE_CLI_SHELL
```

Comment:
Why not simply `ZIGBEE_SHELL` ?

Reply:
This is a bit of legacy from previous SDK - there are many places in code where we have references to `CLI` instead of Zephyr's `SHELL` so this change may be misleading. What do you think about this?